crates/rspack/tests/tree-shaking/ts-target-es5/expected/main.js

```diff
@@ -45,76 +45,19 @@ function _async_to_generator(fn) {
     };
 }
 },
-"../../../../../node_modules/@swc/helpers/esm/_instanceof.js": function (module, exports, __webpack_require__) {
```

Comment:
Not used

Reply:
![image](https://user-images.githubusercontent.com/17974631/233819905-fe3b9bd5-7cde-4a35-b4f6-ef83cce738be.png)